docs/developers/polls.md

```
@@ -0,0 +1,635 @@
+   - Trade-off: Eliminates anonymous participation
+
+2. **Client-Side Session Tracking** (weak mitigation)
+```typescript
+   // Store in sessionStorage after submission
+   sessionStorage.setItem(`poll-${pollId}-submitted`, 'true');
+   // Check before allowing submission
+   if (sessionStorage.getItem(`poll-${pollId}-submitted`)) {
+     // Show "already submitted" message
+   }
+```
+   - ⚠️ Easily bypassed (incognito mode, clearing storage)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)